packages/botonic-dx/sample-config/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "scripts": {
+    "##########  BUILD": "",
+    "build": "tsc",
+    "build-ci": "tsc -b tsconfig.tests.json",
+
+    "##########  LINT": "",
+    "lint": "npm run lint-core -- --fix && npm run prettier",
+    "lint-ci": "npm run lint-core -- -c ../.eslintrc-ci.js && npm run prettier",
+    "lint-core": "eslint_d --cache --quiet  '*.md' './*.js' '*.js{,on}' '{src,tests}/**/*.[jt]{s,sx}'",
+    "prettier": "prettier --write '**/*.md' '**/*.css' ",
```

Comment:
I would add here the `--no-error-on-unmatched-pattern` flag in order to not get an error if no `css` or `md` files are found.

Reply:
good point. added!